addons/medical_treatment/functions/fnc_treatment.sqf

```diff
@@ -179,6 +179,19 @@ if (isArray (_config >> "sounds")) then {
     ];
 };
 
+private _startCallback = getText (_config >> "callbackStart");
+if (isNil _startCallback) then {
```

Comment:
Can it really be `nil`? `getText` has default return of `""` if nothing is found.